decidim-comments/spec/types/comment_type_spec.rb

```diff
@@ -37,6 +37,12 @@ module Comments
           end
           expect(response["replies"]).to_not include("id" => random_comment.id.to_s)
         end
+
+        it "return comment's replies ordered by date" do
+          response_ids = response["replies"].map{|reply| reply["id"].to_i }
+          replies_ids = replies.map(&:id)
+          expect(response_ids).to eq(replies_ids)
```

Comment:
You should reorder response by your criteria so you can test if they are correct.